### ISSUE TITLE
Missing config.h

### ISSUE THREAD

**Issue by KhazAkar** [bert/pcb-plugins#1]
Unable to build - missing config.h in branch master and autotools
Screen attached
![zaznaczenie_004](https://cloud.githubusercontent.com/assets/12693890/7910087/c5bc9084-0851-11e5-9f0c-1a47b953cabf.png)

**Comment by bert** [comment]
Hi KhazAkar,

I'm preparing to integrate autotools in a separate branch.

This way one can generate 'config.h' by means of running './autogen.sh'

This goes well so I will be merging this branch soon.

The thing holding me is that the master branch shows signs of bitrot in the ratsel plug-in, which I will need to fix first.

To be continued.

Kind regards,

Bert Timmerman.

**Comment by KhazAkar** [comment]
I tried method with downloading brach "autotools" and "master". Output is the same as without autotools.
I placed it on pastebin -> http://pastebin.com/QTiradmK

**Comment by bert** [comment]
Hi KhazAkar,

I merged the branch autotools into master.

Please run 'autoreconf' before running './autogen.sh'.

Kind regards,

Bert Timmerman.

**Comment by KhazAkar** [comment]
The same. This is weird,because ./autogen.sh creates that config.h
http://pastebin.com/Cm7fzeUL
My system - Fedora Rawhide(23),I have all needed programs etc to compile & run gEDA and PCB.
And it's not a problem with version of fedora,because on Fedora 22 It's the same.
BTW. Sorry for Polish codes of errors. ;)
Best regards
Damian.

**Comment by bert** [comment]
Hi,

The dirs, files and links in my pcb-plugins directory, so you can compare:

total 636
-rw-rw-r--. 1 bert bert  55204 Jun  2 19:25 aclocal.m4
-rw-rw-r--. 1 bert bert   2493 Mar 20  2014 AUTHORS.plug-ins
-rwxrwxr-x. 1 bert bert   4476 Jun  2 19:03 autogen.sh
drwxr-xr-x. 2 bert bert   4096 Jun  2 19:25 autom4te.cache
-rw-rw-r--. 1 bert bert  34376 Jun  2 19:03 ChangeLog
-rw-rw-r--. 1 bert bert  18541 May 31  2010 ChangeLog.plug-ins
-rw-rw-r--. 1 bert bert   2220 Jun  2 19:31 config.h
-rw-rw-r--. 1 bert bert   2006 Jun  2 19:31 config.h.in
-rw-rw-r--. 1 bert bert  27092 Jun  2 19:31 config.log
-rwxrwxr-x. 1 bert bert  34513 Jun  2 19:31 config.status
-rwxrwxr-x. 1 bert bert 237629 Jun  2 19:31 configure
-rw-rw-r--. 1 bert bert   2639 Jun  2 19:25 configure.ac
lrwxrwxrwx. 1 bert bert     32 Nov 26  2011 depcomp -> /usr/share/automake-1.11/depcomp
drwxrwxr-x. 8 bert bert   4096 Apr 29 05:39 doc
drwxrwxr-x. 3 bert bert   4096 Jun  7  2010 examples
-rw-rw-r--. 1 bert bert  12520 May 30  2010 GPLv1-license.txt
-rw-rw-r--. 1 bert bert  25276 May 30  2010 GPLv2-license.txt
-rw-rw-r--. 1 bert bert  35147 May 30  2010 GPLv3-license.txt
lrwxrwxrwx. 1 bert bert     35 Nov 26  2011 install-sh -> /usr/share/automake-1.11/install-sh
-rw-rw-r--. 1 bert bert  26436 May 30  2010 LGPLv2.1-license.txt
-rw-rw-r--. 1 bert bert  23471 Jun  2 19:31 Makefile
-rw-rw-r--. 1 bert bert    683 Jun  2 19:24 Makefile.am
-rw-rw-r--. 1 bert bert  22919 Jun  2 19:31 Makefile.in
lrwxrwxrwx. 1 bert bert     32 Nov 26  2011 missing -> /usr/share/automake-1.11/missing
-rwxr-xr-x. 1 bert bert   1988 Jun  2 19:31 mkinstalldirs
drwxrwxr-x. 2 bert bert   4096 Jun  2 19:31 po
-rw-rw-r--. 1 bert bert   4352 Mar 20  2014 README.plug-ins
drwxrwxr-x. 3 bert bert   4096 Mar 20  2014 src
-rw-rw-r--. 1 bert bert     23 Jun  2 19:31 stamp-h1
-rw-rw-r--. 1 bert bert     10 Sep 15  2010 stamp-h.in

Kind regards,

Bert Timmerman.

**Comment by KhazAkar** [comment]
[khazakar@dhcppc2 pcb-plugins-master]$ ls -la
razem 672
drwx------   7 khazakar khazakar   4096 06-02 19:40 .
drwxr-xr-x. 13 khazakar khazakar   4096 06-02 19:37 ..
-rw-rw-r--   1 khazakar khazakar  65302 06-02 19:38 aclocal.m4
-rw-r--r--   1 khazakar khazakar   2493 06-02 19:08 AUTHORS.plug-ins
-rwxr-xr-x   1 khazakar khazakar   4476 06-02 19:08 autogen.sh
drwxr-xr-x   2 khazakar khazakar   4096 06-02 19:40 autom4te.cache
-rw-r--r--   1 khazakar khazakar  34376 06-02 19:08 ChangeLog
-rw-r--r--   1 khazakar khazakar  18541 06-02 19:08 ChangeLog.plug-ins
lrwxrwxrwx   1 khazakar khazakar     32 06-02 19:38 compile -> /usr/share/automake-1.15/compile
-rw-rw-r--   1 khazakar khazakar   2220 06-02 19:38 config.h
-rw-rw-r--   1 khazakar khazakar   2006 06-02 19:38 config.h.in
-rw-rw-r--   1 khazakar khazakar  24631 06-02 19:40 config.log
-rwxrwxr-x   1 khazakar khazakar  35045 06-02 19:40 config.status
-rwxrwxr-x   1 khazakar khazakar 247042 06-02 19:38 configure
-rw-r--r--   1 khazakar khazakar   2639 06-02 19:08 configure.ac
drwx------   6 khazakar khazakar   4096 06-02 19:37 doc
drwx------   3 khazakar khazakar   4096 06-02 19:37 examples
-rw-r--r--   1 khazakar khazakar    187 06-02 19:08 .gitignore
-rw-r--r--   1 khazakar khazakar  12520 06-02 19:08 GPLv1-license.txt
-rw-r--r--   1 khazakar khazakar  25276 06-02 19:08 GPLv2-license.txt
-rw-r--r--   1 khazakar khazakar  35147 06-02 19:08 GPLv3-license.txt
lrwxrwxrwx   1 khazakar khazakar     35 06-02 19:38 install-sh -> /usr/share/automake-1.15/install-sh
-rw-r--r--   1 khazakar khazakar  26436 06-02 19:08 LGPLv2.1-license.txt
-rw-rw-r--   1 khazakar khazakar  26373 06-02 19:40 Makefile
-rw-r--r--   1 khazakar khazakar    667 06-02 19:08 Makefile.am
-rw-rw-r--   1 khazakar khazakar  25721 06-02 19:40 Makefile.in
lrwxrwxrwx   1 khazakar khazakar     32 06-02 19:38 missing -> /usr/share/automake-1.15/missing
drwxrwxr-x   2 khazakar khazakar   4096 06-02 19:40 po
-rw-r--r--   1 khazakar khazakar   4352 06-02 19:08 README.plug-ins
drwx------   3 khazakar khazakar   4096 06-02 19:37 src
-rw-rw-r--   1 khazakar khazakar     23 06-02 19:40 stamp-h1
Mine. Weird...
BTW. Changing chmods don't change anything,the same  ;-;
Best regards,
Damian

**Comment by bert** [comment]
Hi Damian,

You do not seem to have mkinstalldirs and stamp-h.in ?

stamp-h.in contains just the string: timestamp

mkinstalldirs is the usual shell script found all over the place:

# mkinstalldirs --- make directory hierarchy

# Author: Noah Friedman friedman@prep.ai.mit.edu

# Created: 1993-05-16

# Public domain

etc.

I don't know why autoreconf works for me on the older (and obsolete for some time) Fedora Release 13 (Goddard) with linux kernel-PAE-2.6.34.9-69.fc13.i686 and Fedora Rawhide-23 not.

Kind regards,

Bert Timmerman.

**Comment by bert** [comment]
Oh well, don't you just love markdown when plain text would have been enough ... sigh

**Comment by bert** [comment]
Hi Damian,

Did you try by invoking:

autoreconf --force

or

autoreconf --install

Kind regards,

Bert Timmerman.

**Comment by KhazAkar** [comment]
Yes,I tried it and it shows the same. It's really weird,because now I have Arch Linux(Fedora died from unknown reason. I'm sending log
http://pastebin.com/bz8BCbtB
I don't know it too,eh :/

**Comment by bert** [comment]
Hi,

My config.h is in http://pastebin.com/HbkFyh1e

Kind regards,

Bert Timmerman

**Comment by KhazAkar** [comment]
Hi ;)
It's mine config.h http://pastebin.com/ZXinwEY1
Best regards,
Damian.
